### PR TITLE
Covered sly7-7 and venusang's contributions to updating documentation (Times 126)

### PR DESCRIPTION
**Pull request by ijlee2**

### CODE REVIEW

File: source/2019-11-29-the-ember-times-issue-126.md [hunk 1]
```diff
@@ -54,13 +55,13 @@ All questions are optional and reaching 100% completion is not required so give
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Latest Updates to Octane Guides 🙏](https://octane-guides-preview.emberjs.com/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Documentation is integral to helping us understand how to use open-source projects like Ember and its addons. Documentation is so common in our lives that we tend to overlook how much work is involved in creating and maintaining one.
```
Comment (amyrlam) [inline source/2019-11-29-the-ember-times-issue-126.md]:
```suggestion
Documentation is integral to helping us understand how to use open source projects like Ember and its addons. Documentation is so common in our lives that we tend to overlook how much work is involved in creating and maintaining one.
```

File: source/2019-11-29-the-ember-times-issue-126.md [hunk 2]
```diff
@@ -54,13 +55,13 @@ All questions are optional and reaching 100% completion is not required so give
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Today, we extend our hearty thanks to [@sly7-7](https://github.com/sly7-7) and [@venusang](https://github.com/venusang). @sly7-7 proofread and updated several code examples in the [Octane Guides](https://octane-guides-preview.emberjs.com/), while @venusang added instructions for using element modifiers in the [Ember.js Octane vs. Classic Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/) (we highly recommend bookmarking this page!).
```
Comment (amyrlam) [inline source/2019-11-29-the-ember-times-issue-126.md:62]:
```suggestion
Today, we extend our hearty thanks to [Sylvain MINA @sly7-7](https://github.com/sly7-7) and [@venusang](https://github.com/venusang). @sly7-7 proofread and updated several code examples in the [Octane Guides](https://octane-guides-preview.emberjs.com/), while @venusang added instructions for using element modifiers in the [Ember.js Octane vs. Classic Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/) (we highly recommend bookmarking this page!).
```

**Review by amyrlam — APPROVED** [review]
👍 love the 🦃 allusions

File: source/2019-11-29-the-ember-times-issue-126.md (hunk 2 above)
Comment (ijlee2) [reply source/2019-11-29-the-ember-times-issue-126.md:62]: Here, I didn't include Sylvain's name, as venusang didn't reveal their name on GitHub. Felt weird to write one person's name but not the other. What do you think?

Comment (amyrlam) [reply source/2019-11-29-the-ember-times-issue-126.md:62]: yeah that's a good point. i think without (as-is) is a-ok!

**Pull request merged**